virtcontainers/kata_agent.go

```diff
@@ -775,12 +777,40 @@ func (k *kataAgent) createContainer(sandbox *Sandbox, c *Container) (p *Process,
 		return nil, err
 	}
 
+	var devInfos []config.DeviceInfo
+	for _, devInfo := range c.config.DeviceInfos {
+		if devInfo.DevType == "e" {
+			filename := filepath.Join(ephemeralPath, filepath.Base(devInfo.ContainerPath))
```

Comment:
If the Base guaranteed to be unique here? If not, I would append this with a randomly generated string to make sure there are no conflicts.

Reply:
Yes, the Base is guaranteed to be unique. In k8s ephemeral volume is defined by, 
```
  volumes:
  - name: cache-volume
    emptyDir:
      medium: "Memory"
```
In the pod yaml. 

The Base comes from `name` of the volume which is unique across the pod (which in the example above would be `cache-volume`)